src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -362,4 +365,8 @@ public boolean getHasError() {
     public void setHasError(boolean value) {
         this.hasError = value;
     }
+
+    public String getIsToBeLoadedInEditMode() {
```

Comment:
I think this can be a boolean too. It should only finally become a string when the JSP echoes it out in HTML.